### DISCUSSION TITLE
nice-article/itmind/sublimetextzcmpjazjcqckyxbxt

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#117] [Announcements]
# nice-article/itmind/sublimetextzcmpjazjcqckyxbxt

Sublime Text 注册码破解安装教程（亲测可用）

https://tobebetterjavaer.com/nice-article/itmind/sublimetextzcmpjazjcqckyxbxt.html

**Comment by skylark0712** [comment]
that license key is no longer valid
